onie-image.conf

```diff
@@ -31,7 +31,10 @@ OUTPUT_ONIE_IMAGE=target/sonic-$TARGET_MACHINE.bin
 OUTPUT_RAW_IMAGE=target/sonic-$TARGET_MACHINE.raw
 
 ### Raw image size in MB
-RAW_IMAGE_DISK_SIZE=8192
+RAW_IMAGE_DISK_SIZE=1024
+
+### Chunk size in MB (0 => single chunk)
+CHUNK_SIZE=8
```

Comment:
default to 0

Reply:
If CHUNK_SIZE=0, the sonic-broadcom.raw will be a 1GB ext4 compressed to ~ 360MB.
Should we have the CHUNK_SIZE option at all to split the raw image into chunks OR it can be done by an offline script ?

Reply:
keep CHUNK_SIZE as an option, I think default to 0 is better.